notebooks/MAST/m_dwarf_tess/m_dwarf_flares_tess.ipynb

```diff
@@ -0,0 +1,1285 @@
+{
```

Comment:
<p>You should briefly describe what's in the other notebook (~1-2 sentences), then put a [link here]. I'll make sure that when the notebooks go live, an appropriate link is added</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/33/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>